README.md

```diff
@@ -95,6 +100,13 @@ We follow a [Gitflow Worklow](https://www.atlassian.com/git/tutorials/comparing-
 
 ![gitflow workflow](https://www.atlassian.com/dam/jcr:b5259cce-6245-49f2-b89b-9871f9ee3fa4/03%20(2).svg)
 
+#### Clear Redis cache
+If you need to clear the Redis cache after deploy, run these commands on your local terminal:
```

Comment:
Can we automate this to happen on build? So when we deploy the cache is flushed?

Reply:
Well that is very annoying. maybe @simaob can help?